Client.py

```diff
@@ -7,22 +7,19 @@ class Client(Network):
     def __init__(self):
         super().__init__()
         self.socket.bind((self.get_ip(), 4005))
-        self.socket.settimeout(2)  # interferes with stopping
-
-        # ip address of 'MODBOT BASE' computer in lab sector A
-        self.server= ("192.168.4.180", 4000)
+        #self.socket.settimeout(4)  # interferes with stopping
 
     def send_data(self, data):
         """ sends json-like nested data containing sensor, accelerometer, etc.
         """
         self.socket.sendto(pickle.dumps(data), self.server)
```

Comment:
@SanjitBasker Would there be benefit in having a recovery timeout loop in here too?

Reply:
I think that if the sever recovers then it would be redundant, just wanted to check my logic

Reply:
^ I think that not only would it be redundant, it would slow us down if both the server and the client are "taking turns sleeping," and it could cause deadlock if the overall frequencies of sleeping/pinging are extremely close (for example, within 1% --> on the order of up to 100 cycles for both the server and client to get a communication established)

Reply:
Agreed